app/views/calculation/income/currentIncome.scala.html

```diff
@@ -50,9 +50,8 @@
             field = currentIncomeForm("amount"),
             question = title,
             hintText = Some(Messages("calc.resident.shares.currentIncome.helpText")),
```

Comment:
Are we missing a currency example here?

Reply:
fixed on [ad05c19](https://github.com/hmrc/cgt-calculator-resident-shares-frontend/pull/184/commits/ad05c195e7c914ece2526ab81468c71cd7e46ec5)